carrie_gz/launch/include/gz_ros_bridge.launch.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
```

Comment:
Why is this launch file and the `spawn_robot.launch.py` launch file in an `include` directory?